craft_providers/lxd/lxd_instance.py

```diff
@@ -513,6 +516,22 @@ def start(self) -> None:
         :raises LXDError: If the instance fails to start.
         """
         logger.info("Starting instance")
+        if self.info().get("Status") == LXDInstanceState.RUNNING.value:
+            if (
+                state := self.config_get("user.craft_providers.status")
+            ) == ProviderInstanceStatus.FINISHED.value:
+                logger.debug("Instance already running but available.")
+                self.config_set(
+                    "user.craft_providers.status", ProviderInstanceStatus.IN_USE.value
+                )
+                self.execute_run(["shutdown", "-c"])
```

Comment:
We could hide this in a `cancel_shutdown()` method or similar, since the command is called more than once.